firmware/arduino/chessboard/motorDriver.ino

```diff
@@ -42,40 +89,68 @@ void moveSpaces(int motor[], int spaces, int dir)
   0         1        2        3
   Step pin, Dir pin, MS1 pin, MS2 pin
 */
-void setScale(int motor[], int scale) 
+
+void moveStraight(int motor[], int startCol, int startRow, int endCol, int endRow)
```

Comment:
Just want to clarify that the messages I'm sending from Pi are in the form xy where x is the row and y is the column. So this swap of parameter ordering may be confusing and require extra care when calling `moveStraight()` to make sure we pass parameters correctly. Might be better to leave in the order it was previously. This comment is irrelevant if you convert the int parameters to a Point before calling this function.